grails-app/services/org/broadinstitute/orsp/NotifyService.groovy

```diff
@@ -197,38 +213,40 @@ class NotifyService implements SendgridSupport, Status  {
             String replyTo) {
         groovyPageRenderer.render(
                 template: "/notify/originalHeaders",
-                model: [to: toAddresses?.join(","),
-                        from: fromAddress,
-                        cc: ccAddresses?.join(","),
-                        bcc: bccAddresses?.join(","),
+                model: [to     : toAddresses?.join(","),
+                        from   : fromAddress,
+                        cc     : ccAddresses?.join(","),
+                        bcc    : bccAddresses?.join(","),
                         replyTo: replyTo,
                         subject: subject])
     }
 
-
     /*
      * Populate the provided view with content.
      */
+
     private String generateMessageContent(
             String view,
             Issue issue,
             User user,
             String issueLink,
             String comment,
             String details,
-            List<String> recipients) {
+            List<String> recipients,
+            Map<String, String> values) {
         if (issue.isFlagSet(IssueExtraProperty.ORSP_ENTERED_FLAG)) {
             details +=
                     "This project is only being sent to ORSP because it has been entered by ORSP. " +
-                    "Normally, the recipients would have been: " + recipients?.join(", ")
+                            "Normally, the recipients would have been: " + recipients?.join(", ")
         }
         String content = groovyPageRenderer.render(
                 view: view,
-                model: [user: user,
-                        issue: issue,
-                        comment: comment,
-                        details: details,
-                        issueLink: issueLink])
+                model: [user     : user,
+                        issue    : issue,
+                        comment  : comment,
+                        details  : details,
+                        issueLink: issueLink,
+                        values: values ])
```

Comment:
nit, if you format the spacing in the lines above this one, then this one should be consistent.